src/daemon/emitter.cc

```diff
@@ -391,6 +406,7 @@ void Emitter::DoLocalExecute(const base::WorkerPool& pool) {
 void Emitter::DoRemoteExecute(const base::WorkerPool& pool, ResolveFn resolver,
                               const ui32 shard) {
   auto conf = this->conf();
+  std::random_device random_device;
```

Comment:
What kind of options do you want?) There're different kind of distributions, but random device doesn't depend on any such thing.
The only option I see is initial seed.